packages/core/src/browser/storage-service.ts

```diff
@@ -83,8 +84,12 @@ export class LocalStorageService implements StorageService {
     }
 
     protected prefix(key: string): string {
-        const pathname = typeof window === 'undefined' ? '' : window.location.pathname;
-        return `theia:${pathname}:${key}`;
+        if (isElectron()) {
+            return `theia:${key}`;
+        } else {
+            const pathname = typeof window === 'undefined' ? '' : window.location.pathname;
+            return `theia:${pathname}:${key}`;
+        }
```

Comment:
I think we can minimize changes by simply having:

```ts
if (isElectron()) {
  return `theia:${key}`
}
const pathname = typeof window === 'undefined' ? '' : window.location.pathname;
return `theia:${pathname}:${key}`;
```

Reply:
Is the objective always to minimize changes, or to maximize readability?  I find it more readable when two equal alternatives, neither one more the exception or the mainline path, are both put inside an if/else.  No big deal though so I have changed it.

Reply:
I think it's a judgement call, my only a suggestion. In this case since it's a simple `if else` I think it's just as readable to do an early check for electron and return right away.